conf/test_mm10.config

```diff
@@ -5,7 +5,8 @@ params {
     genome = 'mm10'
     outdir = "results/test_mm10"
     input = "${projectDir}/assets/samplesheet_test_mm10.csv"
-    contrasts = "${projectDir}/assets/contrasts_test_mm10.yml"
+    contrasts = 'true'
```

Comment:
are boolean values supposed to be quoted in nextflow?

Reply:
we don't need `contrasts` in the config files now that `main.nf` checks for `contrastsheet`, right?